### DISCUSSION TITLE
how to use generated args

### DISCUSSION THREAD

**Discussion by cancng** [MichalLytek/typegraphql-prisma#67] [Q&A] [answered]
First of all, I have opened issue about my problem but I didn't know it was the wrong place for asking help. Sorry about that.

My question is, 

I want to use generated args for my resolver. For example we have admin only resolver and it lists all users. And we want to use `FindManyUserArgs` which includes cursor, distinct, orderBy etc...

I tried create new InputType class and extend `FindManyUserArgs` but couldn't do this.

**Comment by MichalLytek** [comment, marked as answer]
Args are args, not inputs, so you can't use `@InputType` but only `@ArgsType`:

```ts
@ArgsType()
class MyArgs extends FindManyUserArgs {
  @Field()
  someField!: string;
}
```
